extensions/amp-analytics/0.1/events.js

```diff
@@ -1352,7 +1352,12 @@ export class VideoEventTracker extends EventTracker {
         devAssert(isFiniteNumber(normalizedPercentageInt));
         devAssert(normalizedPercentageInt % percentageInterval == 0);
 
-        if (lastPercentage == normalizedPercentageInt) {
+        // Don't trigger if current percentage is the same as
+        // last tirggered percentage
+        if (
+          lastPercentage == normalizedPercentageInt &&
+          percentages.length > 1
```

Comment:
Can we add the test case that passes once the change is in place?

Reply:
e2e test added :) 

There aren't any unit tests that specifically test VideoAnalytics, so I thought creating an e2e file would help in the future as well.